### PR TITLE
[Fix] Transition between v,V,<C-v> is different with original Vim behavior

### PR DESCRIPTION
**Pull request by tyru**
# What this PR does / why we need it

I expect the same behavior as original Vim :)

* `vv` -> normal mode
* `VV` -> normal mode
* `<C-v><C-v>` -> normal mode

* `vV` -> visual line mode
* `v<C-v>` -> visual block mode
* `Vv` -> visual (char) mode
* `V<C-v>` -> visual block mode
* `<C-v>v` -> visual (char) mode
* `<C-v>V` -> visual line mode

# Which issue(s) this PR fixes

I could not find the issue.

### CODE REVIEW

File: test/mode/modeVisual.test.ts [hunk 1]
```diff
@@ -874,4 +874,51 @@ suite('Mode Visual', () => {
...
+      await modeHandler.handleMultipleKeyEvents(['<C-v>', 'v']);
+      assertEqual(modeHandler.currentMode.name, ModeName.Visual);
+    });
+
+    test('<C-v>V will back to visual line mode', async () => {
+      await modeHandler.handleMultipleKeyEvents(['<C-v>', 'V']);
+      assertEqual(modeHandler.currentMode.name, ModeName.VisualLine);
+    });
+
+    test('<C-v><C-v> will back to normal mode', async () => {
```
Comment (xconverge) [inline test/mode/modeVisual.test.ts]: any chance you can switch these to asset that we got into the correct mode as well? Not a big deal but might be a good idea

C-v...assert in visual block, C-v...assert back in normal etc

**Comment by tyru** [comment]
Fixed test and previous commit message a bit :)

**Comment by xconverge** [comment]
Thank you! This one is a no brainer!

**Comment by tyru** [comment]
Thanks! :smile:

**Pull request merged**